### ISSUE TITLE
panic: unknown resource type aws_redshift_security_group

### ISSUE THREAD

**Issue by maresb** [GoogleCloudPlatform/terraformer#1718]
I'm running `terraformer` on Ubuntu 20.04, built from `master` as of c4b7966 running AWS.

Even with `--excludes="aws_redshift_security_group"` I get

```
2023/06/08 15:38:45 Refreshing state... aws_redshift_security_group.tfer--default
panic: unknown resource type aws_redshift_security_group

goroutine 1237 [running]:
github.com/hashicorp/terraform/plugin.(*GRPCProvider).getResourceSchema(0x5ceeda8?, {0x5ccc363, 0x1b})
        ~/go/pkg/mod/github.com/hashicorp/terraform@v0.12.31/plugin/grpc_provider.go:92 +0x10b
github.com/hashicorp/terraform/plugin.(*GRPCProvider).ReadResource(0xc0003c08c0, {{0x5ccc363, 0x1b}, {{{0x651f2c8, 0xc001101d20}}, {0x5510ee0, 0xc002b6dbc0}}, {0x82cc930, 0x0, 0x0}})
        ~/go/pkg/mod/github.com/hashicorp/terraform@v0.12.31/plugin/grpc_provider.go:321 +0x74
github.com/GoogleCloudPlatform/terraformer/terraformutils/providerwrapper.(*ProviderWrapper).Refresh(0xc0003c07e0, 0xc001cbcf50, 0xc001cbcf00)
        ~/repos/terraformer/terraformutils/providerwrapper/provider.go:171 +0x3a9
github.com/GoogleCloudPlatform/terraformer/terraformutils.(*Resource).Refresh(0xc000a03e60, 0x29?)
        ~/repos/terraformer/terraformutils/resource.go:130 +0x4e
github.com/GoogleCloudPlatform/terraformer/terraformutils.RefreshResourceWorker(0x8123aa?, 0xc003bd4b40?, 0xc003bf4a50?)
        ~/repos/terraformer/terraformutils/utils.go:153 +0xad
created by github.com/GoogleCloudPlatform/terraformer/terraformutils.RefreshResources
        ~/repos/terraformer/terraformutils/utils.go:80 +0xe7
```

**Comment by DanielRieske** [comment]
@maresb This is because this resource got deprecated/removed in `5.0.0` of the `terraform-provider-aws`.

I made a PR to remove this resource, in the meantime you can exclude the redshift resources all together by adding `--excludes=redshift` to your command.

**Comment by maresb** [comment]
This was resolved by #1722, thanks!
